src/test/regress/input/hcatalog_lookup.source

```diff
@@ -2,16 +2,9 @@
 -- test hcatalog lookup
 -- --------------------------------------
 
--- Negative test with GUC disabled
-SET hcatalog_enable = false;
-SELECT * from hcatalog.db.t;
-
 SELECT * FROM pxf_get_item_fields('Hive', '*');
```

Comment:
for each of the higher level use case. please add a one line comment on the type of test